deployments/puppet/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## Unreleased
 
+## puppet-v0.14.0
```

Comment:
Please bump the version in [metadata.json](https://github.com/signalfx/splunk-otel-collector/blob/main/deployments/puppet/metadata.json#L3) so that it'll automatically trigger the release when this is merged.